Mathlib/MeasureTheory/Integral/Average.lean

```diff
@@ -211,4 +223,215 @@ theorem set_average_const {s : Set α} (hs₀ : μ s ≠ 0) (hs : μ s ≠ ∞)
     ENNReal.one_toReal, one_smul]
 #align measure_theory.set_average_const MeasureTheory.set_average_const
 
+-- porting note: was `@[simp]` but `simp` can prove it
+theorem integral_average (μ : Measure α) [IsFiniteMeasure μ] (f : α → E) :
+    ∫ _, ⨍ a, f a ∂μ ∂μ = ∫ x, f x ∂μ := by
```

Comment:
Do we know what changed?

Reply:
Possibly, `simp` was able to prove it in Lean 3 too but this lemma was later in the list, so it was applied first.